src/common/options/rgw.yaml.in

```diff
@@ -1019,6 +1019,7 @@ options:
   level: advanced
   desc: Path prefix to be used for accessing RGW RESTful admin API.
   fmt_desc: The entry point for an admin request URL.
+  long_desc: Note that multisite use `admin` hardcoded so this shouldn't be changed in multisite configurations.
```

Comment:
Note that multisite replication requires the value ``admin``, so this option must be left at the default in such deployments.